target/linux/ramips/patches-4.19/0100-disable-staging-driver-but-DMA.patch

```diff
@@ -0,0 +1,17 @@
+--- a/drivers/staging/Makefile
++++ b/drivers/staging/Makefile
+@@ -43,13 +43,7 @@ obj-$(CONFIG_GREYBUS)		+= greybus/
+ obj-$(CONFIG_BCM2835_VCHIQ)	+= vc04_services/
+ obj-$(CONFIG_DRM_VBOXVIDEO)	+= vboxvideo/
+ obj-$(CONFIG_PI433)		+= pi433/
+-obj-$(CONFIG_SOC_MT7621)	+= mt7621-pci/
+-obj-$(CONFIG_SOC_MT7621)	+= mt7621-pinctrl/
+-obj-$(CONFIG_SOC_MT7621)	+= mt7621-spi/
+-obj-$(CONFIG_SOC_MT7621)	+= mt7621-dma/
+-obj-$(CONFIG_SOC_MT7621)	+= mt7621-mmc/
+-obj-$(CONFIG_SOC_MT7621)	+= mt7621-eth/
+-obj-$(CONFIG_SOC_MT7621)	+= mt7621-dts/
++obj-y				+= mt7621-dma/
```

Comment:
Why not keep
`obj-$(CONFIG_SOC_MT7621)	+= mt7621-dma/`
here?

Reply:
Good point.